### PR TITLE
build: add setup.py and setup.cfg

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2e0cx1x 提交打包配置文件 setup.py 与 setup.cfg](https://app.clickup.com/t/2e0cx1x)

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Typing :: Typed
+
+[options]
+packages = find:
+python_requires = >=3.6
+install_requires =
```
Comment (linjiX) [inline setup.cfg:38]: requires here should be the same as "requirements.txt"

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```
Comment (linjiX) [inline setup.cfg:31]: Will it be better to remove 3.10 support until all features are tested in 3.10?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Typing :: Typed
+
+[options]
+packages = find:
+python_requires = >=3.6
+install_requires =
+    numpy >= 1.20.0, <= 1.20.3; python_version >= '3.7'
+    numpy >= 1.16.0; python_version < '3.7'
```
Comment (linjiX) [inline setup.cfg:39]: The numpy requirement is different with requirments.txt?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**